pkg/storage/gcp/getter.go

```diff
@@ -43,20 +43,20 @@ func (s *Storage) GoMod(ctx context.Context, module, version string) ([]byte, er
 	defer sp.Finish()
 	exists, err := s.Exists(ctx, module, version)
 	if err != nil {
-		return nil, errors.E(op, err)
+		return nil, errors.E(op, err, errors.M(module), errors.V(version))
 	}
 	if !exists {
 		return nil, errors.E(op, errors.M(module), errors.V(version), errors.KindNotFound)
 	}
 
 	modReader, err := s.bucket.Open(ctx, config.PackageVersionedName(module, version, "mod"))
 	if err != nil {
-		return nil, errors.E(op, err)
+		return nil, errors.E(op, err, errors.M(module), errors.V(version))
 	}
 	modBytes, err := ioutil.ReadAll(modReader)
 	modReader.Close()
 	if err != nil {
-		return nil, errors.E(op, fmt.Errorf("could not get new reader for mod file: %s", err))
+		return nil, errors.E(op, fmt.Errorf("could not get new reader for mod file: %s", err), errors.M(module), errors.V(version))
```

Comment:
I know this is old but you could probably remove the formatted error for consistency, the other calls no longer do this.